client/gutenberg/extensions/related-posts/edit.jsx

```diff
@@ -17,95 +17,92 @@ export const MAX_POSTS_TO_SHOW = 6;
 
 function PlaceholderPostEdit( props ) {
 	return (
-		<div
+		<ul
 			className="jp-related-posts-i2__post"
 			id={ props.id }
 			aria-labelledby={ props.id + '-heading' }
 		>
-			<strong id={ props.id + '-heading' } className="jp-related-posts-i2__post-link">
-				{ __( 'Preview: Not enough related posts found' ) }
-			</strong>
+			<li className="jp-related-posts-i2__post-link">
+				<a id={ props.id + '-heading' } href={ window.location }>
```

Comment:
Would be good to guard against `window` not existing — this can happen e.g. in our testing setups.

Basically just;
```js
typeof window !== 'undefined'
```